docs/content/guides/author-apps/azure/howto-azure-container-instances/snippets/env.bicep

```diff
@@ -0,0 +1,32 @@
+extension radius
+
+resource env 'Applications.Core/environments@2023-10-01-preview' = {
+  name: 'aci-demo'
```

Comment:
Should we changed this to `aci-dev`? The resources in the screenshot have that as the env name